pkg/sloop/queries/eventquery_test.go

```diff
@@ -91,7 +91,7 @@ func Test_GetEventData_True(t *testing.T) {
 	values := helper_get_params()
 	values[KindParam] = []string{"someKinda"}
 	values[NamespaceParam] = []string{"someNamespace"}
-	values[NameParam] = []string{"someName"}
+	values[NameParam] = []string{"someName.xx"}
```

Comment:
is this test related to ur change ?

Reply:
@kritikapradhan no, a test was failing, this fixed it